### ISSUE TITLE
SlurmConfig does not read options that have hyphens in their name

### ISSUE THREAD

**Issue by jluethi** [fractal-analytics-platform/fractal-server#348]
In the current setup, all our executors only use a single CPU core, even the CPU-high (supposed to take 16) only takes one.

**Comment by tcompa** [comment]
For the record, at UZH this submission script
```
#!/bin/bash

#SBATCH --job-name=test
#SBATCH --partition=main
#SBATCH --nodes=1
#SBATCH --ntasks-per-node=1
#SBATCH --cpus-per-task=4

echo Start
sleep 5
echo End
```
leads to
```
JOBID            USER    ACCOUNT                           NAME    STATE       TIME TIME_LIMIT CPUS MIN_MEMORY NODELIST(REASON) DEPENDENCY
9430760       fractal       root                           test  RUNNING       0:01 7-00:00:00    4     62533M pelkmanslab-slurm-worker-014 
```
Which looks correct.
Then we should just check that the fractal submission scripts look like this one.

Do you observe the same behavior on the cluster you are using?

**Comment by jluethi** [comment]
Will check on the UZH server tomorrow, current tests were on FMI servers. But I thought I also saw it on the UZH cluster.

**Comment by tcompa** [comment]
There is a bug in the way some options are read from the configuration JSON - will report about is soon.

TL;DR
It seems that options which have an hyphen (like `ntasks-per-node`) are not parsed correctly.
